include/misc/byteorder.h

```diff
@@ -1,328 +1,13 @@
- *  unaligned memory location, and convert it to the host endianness.
- *
- *  @param src Location of the little-endian 16-bit integer to get.
- *
- *  @return 16-bit integer in host endianness.
- */
-static inline u16_t sys_get_le16(const u8_t src[2])
-{
-	return ((u16_t)src[1] << 8) | src[0];
-}
-
-/**
- *  @brief Get a 32-bit integer stored in little-endian format.
- *
- *  Get a 32-bit integer, stored in little-endian format in a potentially
- *  unaligned memory location, and convert it to the host endianness.
- *
- *  @param src Location of the little-endian 32-bit integer to get.
- *
- *  @return 32-bit integer in host endianness.
- */
-static inline u32_t sys_get_le32(const u8_t src[4])
-{
-	return ((u32_t)sys_get_le16(&src[2]) << 16) | sys_get_le16(&src[0]);
-}
-
-/**
- *  @brief Get a 64-bit integer stored in little-endian format.
- *
- *  Get a 64-bit integer, stored in little-endian format in a potentially
- *  unaligned memory location, and convert it to the host endianness.
- *
- *  @param src Location of the little-endian 64-bit integer to get.
- *
- *  @return 64-bit integer in host endianness.
- */
-static inline u64_t sys_get_le64(const u8_t src[8])
-{
-	return ((u64_t)sys_get_le32(&src[4]) << 32) | sys_get_le32(&src[0]);
-}
-
-/**
- * @brief Swap one buffer content into another
- *
- * Copy the content of src buffer into dst buffer in reversed order,
- * i.e.: src[n] will be put in dst[end-n]
- * Where n is an index and 'end' the last index in both arrays.
- * The 2 memory pointers must be pointing to different areas, and have
- * a minimum size of given length.
- *
- * @param dst A valid pointer on a memory area where to copy the data in
- * @param src A valid pointer on a memory area where to copy the data from
- * @param length Size of both dst and src memory areas
- */
-static inline void sys_memcpy_swap(void *dst, const void *src, size_t length)
-{
-	u8_t *pdst = (u8_t *)dst;
-	const u8_t *psrc = (const u8_t *)src;
-
-	__ASSERT(((psrc < pdst && (psrc + length) <= pdst) ||
-		  (psrc > pdst && (pdst + length) <= psrc)),
-		 "Source and destination buffers must not overlap");
-
-	psrc += length - 1;
-
-	for (; length > 0; length--) {
-		*pdst++ = *psrc--;
-	}
-}
-
-/**
- * @brief Swap buffer content
- *
- * In-place memory swap, where final content will be reversed.
- * I.e.: buf[n] will be put in buf[end-n]
- * Where n is an index and 'end' the last index of buf.
- *
- * @param buf A valid pointer on a memory area to swap
- * @param length Size of buf memory area
- */
-static inline void sys_mem_swap(void *buf, size_t length)
-{
-	size_t i;
-
-	for (i = 0; i < (length/2); i++) {
-		u8_t tmp = ((u8_t *)buf)[i];
+#warning "This header file has moved, include <sys/byteorder.h> instead."
 
-		((u8_t *)buf)[i] = ((u8_t *)buf)[length - 1 - i];
-		((u8_t *)buf)[length - 1 - i] = tmp;
-	}
-}
+#include <sys/byteorder.h>
 
-#endif /* ZEPHYR_INCLUDE_MISC_BYTEORDER_H_ */
+#endif /* ZEPHYR_INCLUDE_MISC_RING_BUFFER_H_ */
```

Comment:
Not sure what happened here (bad script?) but a lot of the include guards got messed up, it looks like.